### DISCUSSION TITLE
Reset mod settings to default

### DISCUSSION THREAD

**Discussion by Vortigern-The-Grey** [ramensoftware/windhawk#309] [Q&A] [answered]
Looking to disable some mods that I have installed. Is there any way to reset a mod's settings to default? Any help would be greatly appreciated.

**Comment by m417z** [comment]
Currently, the easiest way to reset a mod's settings to their default values is to uninstall and then re-install the mod.

Or if you're asking about resetting to the Windows default behavior, you can disable the mod or remove it.

**Reply by Vortigern-The-Grey** [reply]
The issue I've been having is with the Taskbar Styler mod. When I disable it, I still see a circular outline around the taskbar items.
![image](https://github.com/user-attachments/assets/6430cf04-5ed9-428f-8611-b206cbf711f3)

**Reply by m417z** [reply, marked as answer]
It may happen sometimes, restarting explorer (or logging out) should reset all leftovers.

**Reply by Vortigern-The-Grey** [reply]
Thank you so much!
